.gitignore

```diff
@@ -49,5 +49,10 @@ Thumbs.db
 /lib/build
 /lib/node_modules
 /lib/dist
-
-.nx/
\ No newline at end of file
+/lib/.nyc_output
+/lib/coverage
+! /lib/coverage/branches.csv
+! /lib/coverage/functions.csv
+! /lib/coverage/statements.csv
+! /lib/coverage/lines.csv

```

Comment:
nope, not nécessaire ceux-ci, le check a été simplifié : plutôt que de retenir ce qui est pas couvert, on oblige lib/ à avoir 100% de couverture !